test/run_test.py

```diff
@@ -920,33 +920,38 @@ def run_doctests(test_module, test_directory, options):
     return result
 
 
-def print_log_file(test: str, file_path: str, failed: bool) -> None:
-    num_lines = sum(1 for _ in open(file_path, "rb"))
+def sanitize_file_name(file: str):
+    return file.replace("\\", ".").replace("/", ".").replace(" ", "_")
+
+
+def handle_log_file(
```

Comment:
I think let's keep this function to CI only and leave the log printing as it is when devs use `run_test` locally.  Any thoughts?